app/models/employee.rb

```diff
@@ -316,29 +311,29 @@ def self.adv_search2(p)
       search += ' Category: ' + EmployeeCategory.find(p[:employee_category_id]).name + ', '
     end
 
-    search += 'Joining date:' +  p[:joining_date].to_s + ', ' unless  p[:joining_date].empty?
+    search += 'Joining date:' + p[:joining_date].to_s + ', ' unless p[:joining_date].empty?
 
     if p[:employee_department_id].present?
       search += ' Department: ' + EmployeeDepartment.find(p[:employee_department_id]).name + ', '
-     end
+    end
 
     if p[:employee_position_id].present?
       search += ' Position: ' + EmployeePosition.find(p[:employee_position_id]).name + ', '
-     end
+    end
 
     if p[:employee_grade_id].present?
       search += ' Grade: ' + EmployeeGrade.find(p[:employee_grade_id]).name + ', '
-     end
+    end
 
-    searchs += ' Date of birth: ' +  p[:date_of_birth].to_s + ', ' unless  p[:date_of_birth].empty?
+    searchs << ' Date of birth: ' + p[:date_of_birth].to_s + ', ' unless p[:date_of_birth].empty?
 
-    if p[:status] == 'present'
-      search += 'Staus: Present student'
-    elsif p[:status] == 'former'
-      search += 'Staus: Former student'
-    else
-      search += ' Status: All student'
-    end
+    search += if p[:status] == 'present'
```

Comment:
Refactor condition to the single line.
Have a look at this link- https://saaridev.wordpress.com/2009/11/20/ruby-string-vs/
and try to implement it.